app/scrambler.py

```diff
@@ -1,17 +1,30 @@
+from random import shuffle
+
 from picture import Picture
+from tile import Tile
 
 
 class Scrambler:
-    """Scramble a picture in different ways."""
+    """Scrambler class."""
 
-    def scramble_rows(self, picture: Picture) -> None:
+    @classmethod
+    def scramble_rows(cls, picture: Picture) -> None:
         """Split an Image up into rows an rearranges them."""
-        raise NotImplementedError
+        num_tiles = 8
```

Comment:
Would it make sense to allow a variable number of rows? This works for now, but maybe for the next step that's something we could look into. I think the rest of your code should support already support a variable number of rows.